src/state-indicators.service.js

```diff
@@ -51,11 +53,15 @@ class StateIndicatorsService {
       const location = getLocation(lgas, states, stockCount)
       const locationThresholds = this.thresholdsService.calculateThresholds(location, stockCount)
       const stock = stockCount.stock
+      const products = this.productListService.relevant({byType: true})
 
       const decoratedStock = Object.keys(stock).reduce((decorated, product) => {
         let amount = stock[product]
         let status
         let allocation
+        let selectedProduct = products.find(function (prod) {
+          return prod._id === product
+        })
```

Comment:
Unfortunately `find` won't work unless we include [Babel polyfills](https://babeljs.io/docs/usage/polyfill/), but the polyfills make the module so much bigger that it doesn't seem worth it. Instead what I did was to create my own super simple `find` here https://github.com/fielded/angular-nav-state-indicators/blob/master/src/state-indicators.service.js#L1, you can call it like this https://github.com/fielded/angular-nav-state-indicators/blob/master/src/state-indicators.service.js#L43